ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-release-0.34.yaml

```diff
@@ -3,20 +3,12 @@ base_images:
     name: "4.6"
     namespace: ocp
     tag: base
-  kubevirt-builder:
-    name: kubevirt-builder
-    namespace: ci
-    tag: 30-5.8.1
 build_root:
   image_stream_tag:
-    name: kubevirt-builder
-    namespace: ci
-    tag: 30-5.8.1
+    name: release
+    namespace: openshift
+    tag: golang-1.13
 canonical_go_repository: kubevirt.io/kubevirt
-images:
```

Comment:
After patch is applied, the images stanza is gone. this is intentional?

Reply:
Yes. As we don't need to build anything wrt kubevirt we don't need the kubevirt-builder any more. So we can remove the image section that builds the test binary, and just use the golang image.